src/commands/variations/create.ts

```diff
@@ -59,39 +55,31 @@ export default class CreateVariation extends CreateCommand {
             featureKey = args.feature
         }
 
-        try {
-            const params = await this.populateParametersWithZod(
-                CreateVariationDto,
-                this.prompts, {
-                    key,
-                    name,
-                    headless
-                }
-            )
-
-            let variableAnswers: Record<string, unknown> = {}
-            if (!variables) {
-                const variablesForFeature = await fetchVariables(this.authToken, this.projectKey, featureKey)
-                variableAnswers = await getVariationVariableValuePrompts(
-                    featureKey,
-                    variablesForFeature
-                )
-            }
-
-            const variation = {
-                key: params.key,
-                name: params.name,
-                variables: variables ? JSON.parse(variables) : variableAnswers
+        const params = await this.populateParametersWithZod(
+            CreateVariationDto,
+            this.prompts, {
+                key,
+                name,
+                headless
             }
+        )
 
-            const result = await createVariation(this.authToken, this.projectKey, featureKey, variation)
-            this.writer.showResults(result)
+        let variableAnswers: Record<string, unknown> = {}
+        if (!variables) {
+            const variablesForFeature = await fetchVariables(this.authToken, this.projectKey, featureKey)
+            variableAnswers = await getVariationVariableValuePrompts(
+                featureKey,
+                variablesForFeature
+            )
+        }
 
-        } catch (e) {
-            if (e instanceof ZodError) {
-                this.reportZodValidationErrors(e)
-                return
-            }
+        const variation = {
+            key: params.key,
+            name: params.name,
+            variables: variables ? JSON.parse(variables) : variableAnswers
```

Comment:
I think we should wrap the `JSON.parse` call in a try/catch since someone might enter a malformed string